content/news/2023-03-04-bevy-0.10/index.md

```diff
@@ -0,0 +1,1487 @@
+3. All of the systems are now stored in a single `Schedule` object with no barriers between them?
+4. Systems can belong to any number of system sets, each of which can add their own behavior?
+
+Won't this lead to utter chaos and tedious spaghetti-flavored work to resolve every last ordering ambiguity?
+Many users _liked_ stages, they were helpful for understanding the structure of my app!
+
+Well, I'm glad you asked, rhetorical skeptic. To reduce this chaos (and ease migration), Bevy 0.10 comes with a brand new collection of system sets with the default plugins: [`CoreSet`](https://dev-docs.bevyengine.org/bevy/app/enum.CoreSet.html), [`StartupSet`](https://dev-docs.bevyengine.org/bevy/app/enum.StartupSet.html) and [`RenderSet`](https://dev-docs.bevyengine.org/bevy/render/enum.RenderSet.html). The similarity of their names to [`CoreStage`](https://docs.rs/bevy/0.9.1/bevy/app/enum.CoreStage.html), [`StartupStage`](https://docs.rs/bevy/0.9.1/bevy/app/enum.StartupStage.html) and [`RenderStage`](https://docs.rs/bevy/0.9.1/bevy/render/enum.RenderStage.html) is not a coincidence: there are command flush points between each set, and existing systems have been migrated directly.
+
+Some parts of the stage-centric architecture were appealing: a clear high level structure, coordination on flush points (to reduce excessive bottlenecks) and good default behavior.
+To keep those bits (while excising the frustrating ones), we've introduced the concept of **base sets**, added in [#7466](https://github.com/bevyengine/bevy/pull/7466) by `@cart`. Base sets are system sets, except:
+
+1. Every system can belong to at most one base set.
+2. Systems that do not specify a base set will be added to the default base set for the schedule (if the schedule has one).
+
+```rust
+// You can add new base sets to any built-in ones
+#[derive(Debug, Hash, PartialEq, Eq, Clone, SystemSet)]
+#[system_set(base)]
+enum MyBaseSet {
+    Early,
+    Late,
+}
+
+app
+    // This ends up in CoreSet::Update by default
+    .add_system(no_base_set)
+    // You must use .in_base_set rather than .in_set for explicitness
+    // This is a high-impact decision!
+    .add_system(post_update.in_base_set(CoreSet::PostUpdate))
+    // Look, it works!
+    .add_system(custom_base_set.in_base_set(MyBaseSet::Early))
+    // Ordering your base sets relative to CoreSet is probably wise
+    .configure_set(MyBaseSet::Early.before(CoreSet::Update))
+    .configure_set(MyBaseSet::Late.after(CoreSet::Update));
+```
+
+Pretty simple, but what does this buy us?
+First, it gives you a clear hook to impose, reason about and visualize high level structure to your schedule. Yearning for a linear, stage-like design? Just order your base sets!
+Secondly, it allows Bevy to set good default behavior for systems added by users, without removing their control.
+
+Let me tell you a story, set in a world where all of Mr. Straw Man's points above are true, and no default set is added.
```

Comment:
IIRC, I believe that Mr Straw Man did not survive the editing process, but there is perhaps now a Mr Skeptic?

Reply:
```suggestion
Let me tell you a story, set in a world where all of our rhetorical skeptic's points above are true, and no default set is added.
```